### DISCUSSION TITLE
GetContent changed from v4.9 to v5

### DISCUSSION THREAD

**Discussion by redhorseck** [tinymce/tinymce#7635] [General]
Calling GetContent in v4.9 returned this:
```
<!DOCTYPE html>
<html>
<head>
</head>
<body>
<p>test</p>
</body>
</html>
```

Calling GetContent in v5.10.3 returns this:
`<p>test</p>`

It does not include the `<head>` of the document. Is there a setting to use or a different method to use to get the content including `<head>`? 

Similarly, the code plugin used to include the entire document and now only includes the body. Is it possible some way to include the entire html and not just the body?

**Comment by redhorseck** [comment]
I see that this was achieved with the fullpage plugin. Since that is being deprecated in v6, how is this achieved except by a custom plugin?

**Reply by lostkeys** [reply]
Hi @redhorseck! You should solve this by adding your own HTML document scaffolding around the content exported from TinyMCE. The reason we removed it was it had several frustrating issues, including data loss that was hard to fix. And the plugin was not used much compared to other plugins, which means with our limited resources, there was always a more important bug to address. So the decision was made to drop the full page plugin.

**Reply by redhorseck** [reply]
Ok. Thanks!
